### PR TITLE
📖 Update e2e feature tests triggers

### PR DESCRIPTION
**Pull request by mboukhalfa**
After migrating to prow jenkins operator and merging changes to trigger feature tests from prow this PR update the triggers on readme.md

### CODE REVIEW

**Comment by kashifest** [comment]
/hold
Please fix the tests and also DCO check

File: README.md [hunk 1]
```diff
@@ -175,39 +175,31 @@ without involving pivoting from the bootstrap cluster. To run basic tests use:
...
+- **/test metal3-centos-e2e-feature-test-main** runs e2e feature tests with
+  CAPM3 API version v1beta1 and branch main on CentOS
 
 Release-1.6 branch:
 
-- **/test-ubuntu-e2e-feature-release-1-6** runs e2e feature tests with CAPM3 API
-  version v1beta1 and branch release-1.6 on Ubuntu
-- **/test-centos-e2e-feature-release-1-6** runs e2e feature tests with CAPM3 API
-  version v1beta1 and branch release-1.6 on CentOS
+- **/test metal3-centos-e2e-feature-test-release-1-6** runs e2e feature tests with
```
Comment (tuminoid) [inline README.md:185]: These all are changing ubuntu to centos

Comment (mboukhalfa) [reply README.md]: good catch

**Comment by Sunnatillo** [comment]
/test-centos-e2e-integration-main

testing

**Comment by Sunnatillo** [comment]
> /hold Please fix the tests and also DCO check

DCO issue is fixed.
/unhold

**Comment by mboukhalfa** [comment]
/override 
test-ubuntu-integration-main

**Pull request merged**